src/SayMore/Transcription/Model/TimeTier.cs

```diff
@@ -296,7 +296,7 @@ public BoundaryModificationResult ChangeSegmentsEndBoundary(AnnotationSegment se
 		/// ------------------------------------------------------------------------------------
 		public bool GetIsAcceptableSegmentLength(float start, float end)
 		{
-			return end - start >= Settings.Default.MinimumSegmentLengthInMilliseconds / 1000f;
+			return Math.Round(end * 1000 - start * 1000) >= Settings.Default.MinimumSegmentLengthInMilliseconds;
```

Comment:
What sort of error was occurring? If this is the necessary fix, probably a comment is needed, lest it be put back later by someone wishing to make it more readable.